barretenberg/cpp/src/barretenberg/protogalaxy/protogalaxy_prover_internal.hpp

```diff
@@ -54,6 +55,28 @@ template <class DeciderProvingKeys_> class ProtogalaxyProverInternal {
 
     static constexpr size_t NUM_SUBRELATIONS = DeciderPKs::NUM_SUBRELATIONS;
 
+    inline static FF process_subrelation_evaluations(const RelationEvaluations& evals,
```

Comment:
Interesting note: without this `inline` keyword I was consistently seeing WASM performance slightly worse than baseline.

Reply:
This is despite the fact that there is no corresponding `inline` for the function `RelationUtils::scale_and_batch_elements` that this replaces.

Reply:
hm, and the logic really hasnt changed except refactored a bit